### ISSUE TITLE
Newest Build doesn't work in 1.21.3 or <

### ISSUE THREAD

**Issue by iTrit7** [AdvancedPlugins/Jobs#71]
### Describe the bug

When using the newest Build of the Plugin on Minecraft versions 1.21.3 or earlier it does not work. 

### How to reproduce

Use the latest version of the Plugin on any Minecraft version other than 1.21.4.

### Screenshots / Videos

```
[18:26:07 WARN]: Unexpected exception while parsing console command "jobs"
org.bukkit.command.CommandException: Cannot execute command 'jobs' in plugin AdvancedJobs v1.5.19 - plugin is disabled.
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:37) ~[paper-api-1.21.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:165) ~[paper-api-1.21.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.CraftServer.dispatchCommand(CraftServer.java:1015) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at org.bukkit.craftbukkit.CraftServer.dispatchServerCommand(CraftServer.java:1000) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at net.minecraft.server.dedicated.DedicatedServer.handleConsoleInputs(DedicatedServer.java:505) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at net.minecraft.server.dedicated.DedicatedServer.tickConnection(DedicatedServer.java:454) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1891) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1665) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1330) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:340) ~[paper-1.21.3.jar:1.21.3-81-da71382]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
```

**Comment by Gantz4VT** [comment]
I have the same problem
Yesterday I bought the plugin for my server and when I tried it it didn't work, it came out in red, so I wanted to try a new server with only the plugin in paper 1.21.3 and it didn't work either, I also tried with spigot 1.21.3 and it didn't work either

`[12:45:17] [ServerMain/INFO]: [bootstrap] Running Java 21 (Java HotSpot(TM) 64-Bit Server VM 21.0.4+8-LTS-274; Oracle Corporation null) on Windows 10 10.0 (amd64)
[12:45:17] [ServerMain/INFO]: [bootstrap] Loading Paper 1.21.3-80-master@c2294d7 (2024-12-01T23:55:39Z) for Minecraft 1.21.3
[12:45:18] [ServerMain/INFO]: [PluginInitializerManager] Initializing plugins...
[12:45:18] [ServerMain/INFO]: [PluginInitializerManager] Initialized 1 plugin
[12:45:18] [ServerMain/INFO]: [PluginInitializerManager] Bukkit plugins (1):
 - AdvancedJobs (1.5.19)
[12:45:22] [ServerMain/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[12:45:24] [ServerMain/INFO]: Loaded 1337 recipes
[12:45:24] [ServerMain/INFO]: Loaded 1448 advancements
[12:45:24] [Server thread/INFO]: Starting minecraft server version 1.21.3
[12:45:24] [Server thread/INFO]: Loading properties
[12:45:24] [Server thread/INFO]: This server is running Paper version 1.21.3-80-master@c2294d7 (2024-12-01T23:55:39Z) (Implementing API version 1.21.3-R0.1-SNAPSHOT)
[12:45:25] [Server thread/INFO]: [spark] This server bundles the spark profiler. For more information please visit https://docs.papermc.io/paper/profiling
[12:45:25] [Server thread/INFO]: Server Ping Player Sample Count: 12
[12:45:25] [Server thread/INFO]: Using 4 threads for Netty based IO
[12:45:26] [Server thread/INFO]: [MoonriseCommon] Paper is using 4 worker threads, 1 I/O threads
[12:45:26] [Server thread/INFO]: [ChunkTaskScheduler] Chunk system is using population gen parallelism: true
[12:45:26] [Server thread/INFO]: Default game type: SURVIVAL
[12:45:26] [Server thread/INFO]: Generating keypair
[12:45:26] [Server thread/INFO]: Starting Minecraft server on *:25566
[12:45:26] [Server thread/INFO]: Using default channel type
[12:45:26] [Server thread/INFO]: Paper: Using Java compression from Velocity.
[12:45:26] [Server thread/INFO]: Paper: Using Java cipher from Velocity.
[12:45:26] [Server thread/INFO]: [SpigotLibraryLoader] [AdvancedJobs] Loading 2 libraries... please wait
[12:45:27] [Server thread/INFO]: [SpigotLibraryLoader] [AdvancedJobs] Loaded library C:\Users\Gantzpc2\Desktop\creador e servers mc\servers\Test\libraries\com\zaxxer\HikariCP\3.4.5\HikariCP-3.4.5.jar
[12:45:27] [Server thread/INFO]: [SpigotLibraryLoader] [AdvancedJobs] Loaded library C:\Users\Gantzpc2\Desktop\creador e servers mc\servers\Test\libraries\org\slf4j\slf4j-api\1.7.25\slf4j-api-1.7.25.jar
[12:45:27] [Server thread/INFO]: [SpigotLibraryLoader] [AdvancedJobs] Loaded library C:\Users\Gantzpc2\Desktop\creador e servers mc\servers\Test\libraries\org\apache\commons\commons-math3\3.6.1\commons-math3-3.6.1.jar
[12:45:27] [Server thread/INFO]: [AdvancedJobs] Loading server plugin AdvancedJobs v1.5.19
[12:45:27] [Server thread/INFO]: Server permissions file permissions.yml is empty, ignoring it
[12:45:27] [Server thread/INFO]: Preparing level "world"
[12:45:27] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[12:45:27] [Server thread/INFO]: Preparing spawn area: 0%
[12:45:27] [Paper Common Worker #3/INFO]: [MCTypeRegistry] Initialising converters for DataConverter...
[12:45:28] [Paper Common Worker #3/INFO]: [MCTypeRegistry] Finished initialising converters for DataConverter in 281.9ms
[12:45:28] [Server thread/INFO]: Preparing spawn area: 10%
[12:45:28] [Server thread/INFO]: Preparing spawn area: 10%
[12:45:28] [Server thread/INFO]: Time elapsed: 1243 ms
[12:45:28] [Server thread/INFO]: Preparing start region for dimension minecraft:the_nether
[12:45:28] [Server thread/INFO]: Preparing spawn area: 0%
[12:45:28] [Server thread/INFO]: Time elapsed: 85 ms
[12:45:28] [Server thread/INFO]: Preparing start region for dimension minecraft:the_end
[12:45:28] [Server thread/INFO]: Preparing spawn area: 0%
[12:45:29] [Server thread/INFO]: Time elapsed: 73 ms
[12:45:29] [Server thread/INFO]: [AdvancedJobs] Enabling AdvancedJobs v1.5.19
[12:45:29] [Server thread/INFO]: [AdvancedJobs] Updated lang/en.yml to latest version
[12:45:30] [Server thread/INFO]: [spark] Starting background profiler...
[12:45:30] [Server thread/INFO]: [spark] The async-profiler engine is not supported for your os/arch (windows10/amd64), so the built-in Java engine will be used instead.
[12:45:30] [Server thread/INFO]: Done preparing level "world" (3.041s)
[12:45:30] [Server thread/INFO]: Running delayed init tasks
[12:45:30] [Server thread/INFO]: [AdvancedJobs] Disabling AdvancedJobs v1.5.19
[12:45:30] [Server thread/INFO]: Done (14.867s)! For help, type "help"
[12:46:24] [User Authenticator #0/INFO]: UUID of player Gantz4VT is edf18e39-57fa-4713-98e7-28fc49f963b8
[12:46:25] [Server thread/INFO]: Gantz4VT joined the game
[12:46:25] [Server thread/INFO]: Gantz4VT[/127.136.206.112:53888] logged in with entity id 95 at ([world]-46.085344984256594, 74.0, -57.26427692637551)
[12:46:36] [Server thread/INFO]: Gantz4VT issued server command: /jobs
[12:46:36] [Server thread/ERROR]: Command exception: /jobs
org.bukkit.command.CommandException: Cannot execute command 'jobs' in plugin AdvancedJobs v1.5.19 - plugin is disabled.
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:37) ~[paper-api-1.21.3-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.command.brigadier.bukkit.BukkitCommandNode$BukkitBrigCommand.run(BukkitCommandNode.java:82) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at com.mojang.brigadier.context.ContextChain.runExecutable(ContextChain.java:73) ~[brigadier-1.3.10.jar:?]
	at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:31) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:19) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.execution.UnboundEntryAction.lambda$bind$0(UnboundEntryAction.java:8) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.execution.CommandQueueEntry.execute(CommandQueueEntry.java:5) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.execution.ExecutionContext.runCommandQueue(ExecutionContext.java:103) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.Commands.executeCommandInContext(Commands.java:448) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.Commands.performCommand(Commands.java:355) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.Commands.performCommand(Commands.java:342) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.Commands.performCommand(Commands.java:337) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.performUnsignedChatCommand(ServerGamePacketListenerImpl.java:2241) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleChatCommand$11(ServerGamePacketListenerImpl.java:2215) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:154) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1578) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:207) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:128) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1555) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1548) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:138) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1499) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1509) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1345) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:340) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
`

**Comment by Gantz4VT** [comment]
I have the same problem

Yesterday I bought the plugin for my server and when I tried it it didn't work, it came out in red, so I wanted to try a new server with only the plugin in paper 1.21.3 and it didn't work either, I also tried with spigot 1.21.3 and it didn't work either

`[12:45:17] [ServerMain/INFO]: [bootstrap] Running Java 21 (Java HotSpot(TM) 64-Bit Server VM 21.0.4+8-LTS-274; Oracle Corporation null) on Windows 10 10.0 (amd64)
[12:45:17] [ServerMain/INFO]: [bootstrap] Loading Paper 1.21.3-80-master@c2294d7 (2024-12-01T23:55:39Z) for Minecraft 1.21.3
[12:45:18] [ServerMain/INFO]: [PluginInitializerManager] Initializing plugins...
[12:45:18] [ServerMain/INFO]: [PluginInitializerManager] Initialized 1 plugin
[12:45:18] [ServerMain/INFO]: [PluginInitializerManager] Bukkit plugins (1):
 - AdvancedJobs (1.5.19)
[12:45:22] [ServerMain/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[12:45:24] [ServerMain/INFO]: Loaded 1337 recipes
[12:45:24] [ServerMain/INFO]: Loaded 1448 advancements
[12:45:24] [Server thread/INFO]: Starting minecraft server version 1.21.3
[12:45:24] [Server thread/INFO]: Loading properties
[12:45:24] [Server thread/INFO]: This server is running Paper version 1.21.3-80-master@c2294d7 (2024-12-01T23:55:39Z) (Implementing API version 1.21.3-R0.1-SNAPSHOT)
[12:45:25] [Server thread/INFO]: [spark] This server bundles the spark profiler. For more information please visit https://docs.papermc.io/paper/profiling
[12:45:25] [Server thread/INFO]: Server Ping Player Sample Count: 12
[12:45:25] [Server thread/INFO]: Using 4 threads for Netty based IO
[12:45:26] [Server thread/INFO]: [MoonriseCommon] Paper is using 4 worker threads, 1 I/O threads
[12:45:26] [Server thread/INFO]: [ChunkTaskScheduler] Chunk system is using population gen parallelism: true
[12:45:26] [Server thread/INFO]: Default game type: SURVIVAL
[12:45:26] [Server thread/INFO]: Generating keypair
[12:45:26] [Server thread/INFO]: Starting Minecraft server on *:25566
[12:45:26] [Server thread/INFO]: Using default channel type
[12:45:26] [Server thread/INFO]: Paper: Using Java compression from Velocity.
[12:45:26] [Server thread/INFO]: Paper: Using Java cipher from Velocity.
[12:45:26] [Server thread/INFO]: [SpigotLibraryLoader] [AdvancedJobs] Loading 2 libraries... please wait
[12:45:27] [Server thread/INFO]: [SpigotLibraryLoader] [AdvancedJobs] Loaded library C:\Users\Gantzpc2\Desktop\creador e servers mc\servers\Test\libraries\com\zaxxer\HikariCP\3.4.5\HikariCP-3.4.5.jar
[12:45:27] [Server thread/INFO]: [SpigotLibraryLoader] [AdvancedJobs] Loaded library C:\Users\Gantzpc2\Desktop\creador e servers mc\servers\Test\libraries\org\slf4j\slf4j-api\1.7.25\slf4j-api-1.7.25.jar
[12:45:27] [Server thread/INFO]: [SpigotLibraryLoader] [AdvancedJobs] Loaded library C:\Users\Gantzpc2\Desktop\creador e servers mc\servers\Test\libraries\org\apache\commons\commons-math3\3.6.1\commons-math3-3.6.1.jar
[12:45:27] [Server thread/INFO]: [AdvancedJobs] Loading server plugin AdvancedJobs v1.5.19
[12:45:27] [Server thread/INFO]: Server permissions file permissions.yml is empty, ignoring it
[12:45:27] [Server thread/INFO]: Preparing level "world"
[12:45:27] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[12:45:27] [Server thread/INFO]: Preparing spawn area: 0%
[12:45:27] [Paper Common Worker #3/INFO]: [MCTypeRegistry] Initialising converters for DataConverter...
[12:45:28] [Paper Common Worker #3/INFO]: [MCTypeRegistry] Finished initialising converters for DataConverter in 281.9ms
[12:45:28] [Server thread/INFO]: Preparing spawn area: 10%
[12:45:28] [Server thread/INFO]: Preparing spawn area: 10%
[12:45:28] [Server thread/INFO]: Time elapsed: 1243 ms
[12:45:28] [Server thread/INFO]: Preparing start region for dimension minecraft:the_nether
[12:45:28] [Server thread/INFO]: Preparing spawn area: 0%
[12:45:28] [Server thread/INFO]: Time elapsed: 85 ms
[12:45:28] [Server thread/INFO]: Preparing start region for dimension minecraft:the_end
[12:45:28] [Server thread/INFO]: Preparing spawn area: 0%
[12:45:29] [Server thread/INFO]: Time elapsed: 73 ms
[12:45:29] [Server thread/INFO]: [AdvancedJobs] Enabling AdvancedJobs v1.5.19
[12:45:29] [Server thread/INFO]: [AdvancedJobs] Updated lang/en.yml to latest version
[12:45:30] [Server thread/INFO]: [spark] Starting background profiler...
[12:45:30] [Server thread/INFO]: [spark] The async-profiler engine is not supported for your os/arch (windows10/amd64), so the built-in Java engine will be used instead.
[12:45:30] [Server thread/INFO]: Done preparing level "world" (3.041s)
[12:45:30] [Server thread/INFO]: Running delayed init tasks
[12:45:30] [Server thread/INFO]: [AdvancedJobs] Disabling AdvancedJobs v1.5.19
[12:45:30] [Server thread/INFO]: Done (14.867s)! For help, type "help"
[12:46:24] [User Authenticator #0/INFO]: UUID of player Gantz4VT is edf18e39-57fa-4713-98e7-28fc49fdfgfdg
[12:46:25] [Server thread/INFO]: Gantz4VT joined the game
[12:46:25] [Server thread/INFO]: Gantz4VT[/***.***.***.***:53888] logged in with entity id 95 at ([world]-fd85344984256594, 74.0, -75.26427692637551)
[12:46:36] [Server thread/INFO]: Gantz4VT issued server command: /jobs
[12:46:36] [Server thread/ERROR]: Command exception: /jobs
org.bukkit.command.CommandException: Cannot execute command 'jobs' in plugin AdvancedJobs v1.5.19 - plugin is disabled.
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:37) ~[paper-api-1.21.3-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.command.brigadier.bukkit.BukkitCommandNode$BukkitBrigCommand.run(BukkitCommandNode.java:82) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at com.mojang.brigadier.context.ContextChain.runExecutable(ContextChain.java:73) ~[brigadier-1.3.10.jar:?]
	at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:31) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:19) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.execution.UnboundEntryAction.lambda$bind$0(UnboundEntryAction.java:8) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.execution.CommandQueueEntry.execute(CommandQueueEntry.java:5) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.execution.ExecutionContext.runCommandQueue(ExecutionContext.java:103) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.Commands.executeCommandInContext(Commands.java:448) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.Commands.performCommand(Commands.java:355) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.Commands.performCommand(Commands.java:342) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.commands.Commands.performCommand(Commands.java:337) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.performUnsignedChatCommand(ServerGamePacketListenerImpl.java:2241) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleChatCommand$11(ServerGamePacketListenerImpl.java:2215) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:154) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1578) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:207) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:128) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1555) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1548) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:138) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1499) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1509) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1345) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:340) ~[paper-1.21.3.jar:1.21.3-80-c2294d7]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]`

**Comment by KPGTB** [comment]
Hey! Sorry for this problem. It seems that there was error during plugin compilation which caused this problem. I notified @GC-spigot  - he will recompile & reupload plugin, so fix will be available soon

**Comment by KPGTB** [comment]
Fixed in 1.6.0
